src/settings.js

```diff
@@ -2188,4 +2180,5 @@ var LEGACY_SETTINGS = [
   ['LIBRARY_DEPS_TO_AUTOEXPORT', [[]], 'No longer needed'],
   ['EMIT_EMSCRIPTEN_METADATA', [0], 'No longer supported'],
   ['SHELL_FILE', [''], 'No longer supported'],
+  ['LLD_REPORT_UNDEFINED', [1], 'No longer supported'],
```

Comment:
```suggestion
  ['LLD_REPORT_UNDEFINED', [1], 'Enabled all the time'],
```

Reply:
Actually I'm not sure if this is better. It already says `1` which I guess means 1 is the default, so what is no longer supported is to disable it?